presto/base/postgres-dc.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: presto-postgres
+  labels:
+    app: presto
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    deploymentconfig: presto-postgres
+  template:
+    metadata:
+      labels:
+        deploymentconfig: presto-postgres
+    spec:
+      containers:
+        - name: postgresql
+          image: "quay.io/internaldatahub/postgresql-96-centos7:9.6"
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 5432
+              protocol: TCP
+          readinessProbe:
+            timeoutSeconds: 1
+            initialDelaySeconds: 5
+            exec:
+              command:
+                - "/usr/libexec/check-container"
+          livenessProbe:
+            timeoutSeconds: 10
+            initialDelaySeconds: 120
+            exec:
+              command:
+                - "/usr/libexec/check-container"
+                - "--live"
+          env:
+            - name: POSTGRESQL_USER
+              valueFrom:
+                secretKeyRef:
+                  name: postgres-secret
+                  key: database-user
+            - name: POSTGRESQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: postgres-secret
+                  key: database-password
+            - name: POSTGRESQL_DATABASE
+              valueFrom:
+                secretKeyRef:
+                  name: postgres-secret
+                  key: database-name
+          resources:
+            limits:
+              memory: "512Mi"
```

Comment:
Minimum 1Gi for PostgreSQL is recommended. Also, please specify both requests and limits.

Try use these values:

Request:
* 1 CPU
* 1Gi Memory

Limit:
* 2 CPUs
* 2Gi Memory

Reply:
@rimolive can you review again?